backend/app/libs/tasks/get_assets.py

```diff
@@ -65,11 +76,17 @@ def reload_treasuries_stats():
     start = start_date.isoformat()[:10]
     end = end_date.isoformat()[:10]
 
-    treasuries = retrieve_treasuries_metadata(db)
+    treasuries = (
+        troublesome_treasuries
+        if troublesome_treasuries
+        else retrieve_treasuries_metadata(db)
+    )
     if not treasuries:
         store_treasuries_metadata(db, get_treasury_list())
         treasuries = retrieve_treasuries_metadata(db)
 
+    candidate_troublesome_treasuries: list[tuple[str, int]] = []
```

Comment:
This could also be a set, see my other comment.

Reply:
Okay, both these concerns have been noted. Sets are now being used to store troublesome treasury metadata.

I always left the `chain_id` in the Whip logic in case we eventually start looking at treasuries from other chains such as Gnosis or Optimism, but if you think we should scrap it then we can do so.